### PR TITLE
Upgrade dependencies

### PR DESCRIPTION
**Pull request by sandersky**
**This project uses [semver](http://semver.org), please check the scope of this pr:**

- [ ] #patch# - backwards-compatible bug fix
- [x] #minor# - adding functionality in a backwards-compatible manner
- [ ] #major# - incompatible API change

# CHANGELOG

* **Removed** files from npm package that aren't necessary (all of the various config files).
* **Updated** dependencies to latest versions.

### CODE REVIEW

**Comment by sandersky** [comment]
:+1:

**Comment by sandersky** [comment]
:+1:

**Comment by sandersky** [comment]
:+1:

[Approved with PullApprove](https://pullapprove.com/ciena-blueplanet/ember-test-utils/pull-request/23/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=ciena-blueplanet/ember-test-utils)

File: bower.json [hunk 1]
```diff
@@ -1,18 +1,16 @@
...
     "ember-mocha": "~0.8.11",
     "Faker": "3.0.1",
-    "lodash": "~4.5.0",
     "pretender": "~0.12.0",
     "sinon-chai": "^2.8.0",
     "chai-jquery": "^2.0.1"
   },
   "resolutions": {
-    "ember": "~2.8.0"
+    "ember": "^2.10.2"
```
Comment (job13er) [inline bower.json:14]: I thought we were sticking with 2.8 for a while?

File: README.md [hunk 2]
```diff
@@ -1,15 +1,20 @@
...
-[ci-url]: https://travis-ci.org/ciena-blueplanet/ember-test-utils
+# ember-test-utils
 
-[cov-img]: https://img.shields.io/coveralls/ciena-blueplanet/ember-test-utils.svg "Coveralls Code Coverage"
-[cov-url]: https://coveralls.io/github/ciena-blueplanet/ember-test-utils
+Ember testing utilities.
 
-[npm-img]: https://img.shields.io/npm/v/ember-test-utils.svg "NPM Version"
-[npm-url]: https://www.npmjs.com/package/ember-test-utils
+###### Dependencies
```
Comment (job13er) [inline README.md:5]: I never really liked these sections, they take up so much vertical space. Not sure of a better idea though, except maybe cutting out some of the badges...

File: bower.json (hunk 1 above)
Comment (sandersky) [reply bower.json:14]: It stills works with Ember 2.8 and I added a CI build against 2.8 specifically. The only way we'll ever make it to Ember 2.10 is to start ensuring our various addons work in 2.10. I've already ensured all of our addons in `ciena-blueplanet` work in Ember 2.10 and have started the process on `ember-frost-core`. My goal is to try to get everything working in Ember 2.10 without making any major changes, in an effort to make 2.10 viable in our apps using LTS.

File: README.md (hunk 2 above)
Comment (sandersky) [reply README.md:5]: I personally prefer them take up more vertical space than be crammed in a single line. Open to better proposals though.

**Pull request merged**